<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightAssignEditDestinationDialog</class>
 <widget class="QDialog" name="FlightAssignEditDestinationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lAirdrome">
       <property name="text">
        <string>Аэродром:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QComboBox" name="cbAirdrome"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lSumm">
       <property name="text">
        <string>Требуемая сумма:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QSpinBox" name="sbSumm"/>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QLabel" name="lQuantity">
       <property name="text">
        <string>рублей</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lNumber">
       <property name="text">
        <string>Порядковый номер: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLabel" name="lNumberValue">
       <property name="text">
        <string>номер</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lInfo">
     <property name="text">
      <string>Дополнительная информация:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="teInfo"/>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
